chaos-workers/chaos-experiments/scripts/utils.sh

```diff
@@ -14,19 +14,30 @@ function getNamespace()
  echo "$namespace"
 }
 
-function getLabel() {
+function getBrokerLabels() {
   if [ "${CHAOS_SETUP}" == "cloud" ]; then
-    echo "app.kubernetes.io/app=zeebe"
-  else
-    echo "app.kubernetes.io/component=broker"
+    # For backwards compatability the brokers kept the gateway labels, for a statefulset the labels are not modifiable
+    echo "-l app.kubernetes.io/app=zeebe -l app.kubernetes.io/component=gateway"
```

Comment:
I didn't fully understand this comment, until I read line 29
> To still be able to distinguish the standalone gateway with the broker, the gateway got a new label.

🔧 Please duplicate that line to the comment on line 19, so that it's clear what is going on.